apps/vr-tests-react-components/src/stories/Spinner.stories.tsx

```diff
@@ -12,7 +13,11 @@ const InvertedWrapper: React.FC = ({ children }) => {
 };
 
 storiesOf('Spinner converged', module)
-  .addDecorator(TestWrapperDecoratorNoAnimation)
+  .addDecorator(TestWrapperDecorator)
+  .addDecorator(PauseAnimationDecorator)
+  .addDecorator(story => (
+    <StoryWright steps={new Steps().snapshot('default', { cropTo: '.testWrapper' }).end()}>{story()}</StoryWright>
+  ))
```

Comment:
I don't see any Spinner screenshots changed in the VR approval app -- I'd expect that they would be changed by this?